### ISSUE TITLE
Retrain to boost performance

### ISSUE THREAD

**Issue by lianxxx** [xtudbxk/DSRG-tensorflow#15]
Hi,

Have you tried the retrain step, which can boost the performance by around 1.5%, as mentioned in the original DSRG paper? 

I tried to implement the retrain step based on your code, but I got worse results compared to those without retrain. Sad..............................................................................................................

**Comment by xtudbxk** [comment]
The retrain step indeed improves the performance in my experiment. Maybe you can predict all the labels and use codes from the project aims to implement the origin deeplabv2.

**Comment by lianxxx** [comment]
Was your retrain model initialized with the weights from the pre-training with DSRG? Could you please kindly provide your re-training hype-parameters: init_learning_rate, learning rate policy, batch_size, number of training epoches? Btw, how much improvements did you get from retraining? Looking forward to your reply. Thank you very much!!!

**Comment by xtudbxk** [comment]
Yes, it was. And it seems I didn't change the hyper-parameters for the retrain step. And in my experiments which are not same with DSRG, the retrain steps seems  to improve about 1% in miou.
